src/app/pages/datasets/components/dataset-management/dataset-management.component.ts

```diff
@@ -304,9 +305,10 @@ export class DatasetsManagementComponent implements OnInit, AfterViewInit, OnDes
   private listenForDatasetScrolling(): void {
     this.subscription.add(
       this.scrollSubject
-        .pipe(distinctUntilChanged(), untilDestroyed(this))
+        .pipe(debounceTime(5), untilDestroyed(this))
```

Comment:
Noticed a hard-coded delay. I'd suggested to move it to the named constant. And perhaps the delay length could be changed to 16ms (60 FPS) without degradation of smoothness of scrolling, but not sure.